news/2019-12-27-osu-mapping-olympiad-3-results.md

```diff
@@ -0,0 +1,51 @@
+
+In **first place** is **[PianoLuigi](https://osu.ppy.sh/users/9665915)** with a total score of **133/150**! Their entry scored the highest in an impressive 5 of the 6 categories and was the highest scoring entry from judges Asherz007 and Leniane! This was an entry that stood high above the rest and is well deserving of the top spot for this contest. Congratulations!
+
+In **second place** is **[Kawawa](https://osu.ppy.sh/users/4647754)** with a total score of **120/150**. With excellent scores across each category and the highest score from judge Tailsdk, Kawawa's entry was very well executed.
+
+And rounding out the podium in **third place** is **[\_VianK\_](https://osu.ppy.sh/users/6889055)** with a total score of **109/150**. Strong scores from each judge and in the technique and hitsounding categories helped push their entry into third place.
+
+You can see all of the scores and comments from the judges via [this spreadsheet](https://docs.google.com/spreadsheets/d/1f1JEhs8iAbcWZppB2JJ25k4qerFE_RAOZCN_LYzo4NQ/edit?usp=sharing), and you can [click here to download all of the entries](https://drive.google.com/open?id=1fbVsSd0R3TtpMn5F3FYKEiVcRd-njXBz) via Google Drive.
+
+Congratulations to all of the top three finishers! Here is what each person will receive as their reward (as a reminder, 6 Olympiad Points are needed for one badge level):
+
+- **First place: PianoLuigi:** 4 months of osu!supporter, 6 Olympiad Points (awards "Mapping Olympian" badge), *Elite Mapper* title
+- **Second place: Kawawa:** 3 months of osu!supporter, 4 Olympiad Points
+- **Third place: \_VianK\_:** 2 months of osu!supporter, 2 Olympiad Points
+
+As a bonus for winning first place, PianoLuigi's badge description will include the song title (*Scavenger*) they placed first in!
+
+You can also visit the [listing of all accumulated Olympiad Points](https://docs.google.com/spreadsheets/d/1_gIDJwTOgIhGr2h4069-r1C-2GHuV--5wtwPywzYsz8/edit), which is updated after each contest.
+
+We would like to give a big thank you to [Asherz007](https://osu.ppy.sh/users/9014047), [Leniane](https://osu.ppy.sh/users/7138602), and [Tailsdk](https://osu.ppy.sh/users/6751666) for stepping up to judge the entries. As with the osu!taiko contest, we unfortunately ran into issues with getting judging completed for this contest. It was a huge help for these three to come in and judge the entries so that we could finish the contest. We could not have done it without them. Please note that while Asherz007 helped organize the beginning parts of this contest, he did not have access to the identities of the entries as the anonymization process was handled by Chaos.
+
+Thank you to all of the participants and congratulations again to [PianoLuigi](https://osu.ppy.sh/users/9665915)! Be on the lookout for another osu!catch Olympiad contest coming soon, and an osu!standard contest on the horizon!
```

Comment:
are the contests coming soon, or the results?

Reply:
The contests are going to be announced soon